.vib/gitea/vib-publish.json

```diff
@@ -0,0 +1,85 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/gitea"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/gitea"
+        },
+        "runtime_parameters": "c2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHA6IDgwCmNvbnRhaW5lclBvcnRzOgogIGh0dHA6IDgxMjMKY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogIHJ1bkFzVXNlcjogMTAwMgpwb2RTZWN1cml0eUNvbnRleHQ6CiAgZnNHcm91cDogMTAwMgphZG1pblBhc3N3b3JkOiAiYml0bmFtaSExMjM0IgoKcG9zdGdyZXNxbDoKICBhdXRoOgogICAgdXNlcm5hbWU6IGJuX3ZpYl9naXRlYQogICAgZGF0YWJhc2U6IGJpdG5hbWlfdmliX2dpdGVhCiAgcHJpbWFyeToKICAgIHNlcnZpY2U6CiAgICAgIHBvcnRzOgogICAgICAgIHBvc3RncmVzcWw6IDkzOTMK",
```

Comment:
With the idea to add any parameter necessary for the correct execution of the tests, I would also add the following one:

- `podSecurityContext.enabled=true`
- `containerSecurityContext.enabled=true`
- `postgresql.enabled=true`